### ISSUE TITLE
Timers should be usable as counters

### ISSUE THREAD

**Issue by paavohuhtala** [OpenSAGE/OpenSAGE#36]
I was debugging a map script error in the shell map, when I encountered the following script:

```
*** IF ***
    Counter 'Reset Camera' IS Equal To  1 
*** THEN ***
  Enable Script 'Follow Circle to see battlefield'.
```

'Reset Camera' is a timer, not a counter. A bit of googling led me to the [World Builder manual for BFME](http://largedownloads.ea.com/pub/misc/bfme/WorldBuilder_manual.pdf), which states:
>Timers always count down from the start time you set. A timer can be used as a counter, 
too. You can perform all of the operations on a timer that you can perform on a counter.

**Comment by paavohuhtala** [comment]
Implemented in #37

**Comment by tgjones** [comment]
Fixed in #37.
